### DISCUSSION TITLE
no layoutfile

### DISCUSSION THREAD

**Discussion by echoandzoey** [kdevo/osprey-delight#51] [Q&A]
I have try all the solutions to solve the problem but not work, it can just find and layout file,how should i do ?

D:\zoeyPortfoilos\zoeysite.github.io\zoeysite>hugo serve
Watching for changes in D:\zoeyPortfoilos\zoeysite.github.io\zoeysite\{archetypes,assets,content,data,i18n,layouts,static}
Watching for config changes in D:\zoeyPortfoilos\zoeysite.github.io\zoeysite\hugo.yaml, D:\zoeyPortfoilos\zoeysite.github.io\zoeysite\go.mod
Start building sites …
hugo v0.121.1-00b46fed8e47f7bb0a85d7cfc2d9f1356379b740+extended windows/amd64 BuildDate=2023-12-08T08:47:45Z VendorInfo=gohugoio

WARN  found no layout file for "html" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN  found no layout file for "html" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

                   | EN
-------------------+-----
  Pages            |  3
  Paginator pages  |  0
  Non-page files   |  0
  Static files     |  0
  Processed images |  0
  Aliases          |  0
  Sitemaps         |  1
  Cleaned          |  0

Built in 5 ms
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender

**Comment by kdevo** [comment]
This is not an issue of the theme, seems like a configuration issue -> converted to discussion.

How does your config.yaml look like? Have you tried if the `exampleSite` works for you? If yes, it can help to use it as a starter (and refer to README instructions).
